app/server/lib/coreCreator.ts

```diff
@@ -13,6 +18,24 @@ export const makeCoreCreator = () => makeSimpleCreator({
       create: configureMinIOExternalStorage,
     },
   ],
+  attachmentStoreOptions: [
+    {
+      name: 'minio',
+      isAvailable: async () => checkMinIOExternalStorage() !== undefined,
+      // TODO - Move this to another file
```

Comment:
What do you envision should be done here? Which file?

Reply:
....Honestly not sure. At the time I wrote this comment, it was likely we'd have a `MinIOAttachmentStore` file. But that got generalised into `ExternalStorageAttachmentStore`.

 It feels long for where it is. Might be just worth moving it to the end of this file and referencing it, to keep the coreCreator definition easy to read. 
 
 What do you think?

Reply:
It's pretty short, but if you want to move it to the bottom of the file, that's fine too.

It would also be fine to get rid of the TODO comment. Actually, that's what I really want, to reduce the number of TODOs by one.